src/elm/Page/Community/Objectives.elm

```diff
@@ -244,91 +233,93 @@ viewAction ({ shared } as loggedIn) model objectiveId action =
+                            [ if action.usages > 0 then
+                                p [ classList [ ( "text-red", action.usagesLeft == 0 ) ] ]
+                                    [ text (tr "community.actions.usages" [ ( "usages", usages ), ( "usagesLeft", usagesLeft ) ]) ]
+
+                              else
                                 text ""
+                            , case action.deadline of
+                                Just _ ->
+                                    p [ classList [ ( "text-red", pastDeadline ) ] ] [ text deadlineStr ]
+
+                                Nothing ->
+                                    text ""
+                            ]
                         ]
-                    ]
-            , div [ class "mx-2 mb-2" ]
-                [ if action.isCompleted then
-                    div [ class "tag bg-green" ] [ text_ "community.actions.completed" ]
+                , div [ class "mx-2 mb-2" ]
+                    [ if action.isCompleted then
+                        div [ class "tag bg-green" ] [ text_ "community.actions.completed" ]
 
-                  else if isClosed then
-                    div [ class "tag bg-gray-500 text-red" ] [ text_ "community.actions.closed" ]
+                      else if isClosed then
+                        div [ class "tag bg-gray-500 text-red" ] [ text_ "community.actions.closed" ]
 
-                  else
-                    text ""
+                      else
+                        text ""
+                    ]
                 ]
-            ]
-        , div [ class "flex justify-between items-end py-8 flex-col" ]
-            [ div [ class "w-full" ]
-                [ p [ class "input-label mb-4" ] [ text_ "community.actions.verifiers" ]
-                , if validationType == "AUTOMATIC" then
-                    div [ class "flex items-center" ]
-                        [ p [ class "text-body" ] [ text_ "community.actions.automatic_analyzers" ]
-                        , Icons.exclamation "ml-2"
-                        ]
+            , div [ class "flex flex-wrap justify-between items-end" ]
+                [ div [ class "w-full sm:w-4/5" ]
+                    [ p [ class "input-label mb-4" ] [ text_ "community.actions.verifiers" ]
+                    , if validationType == "AUTOMATIC" then
+                        div [ class "flex items-center" ]
+                            [ p [ class "text-body" ] [ text_ "community.actions.automatic_analyzers" ]
+                            , Icons.exclamation "ml-2"
+                            ]
 
-                  else
-                    div [ class "flex overflow-scroll -mx-2" ]
-                        (List.map
-                            (\u ->
-                                div [ class "mx-2" ]
-                                    [ Profile.view shared loggedIn.accountName u ]
+                      else
+                        div [ class "flex mr-2 overflow-x-scroll" ]
```

Comment:
I think it's better to use `overflow-x-auto` here because `overflow-x-scroll` always shows the scroll (grey area under verifiers on the screenshot) even if there are only few verifiers. It looks odd:
![image](https://user-images.githubusercontent.com/140053/87954776-7d66c580-cab5-11ea-9742-47d2a22e1d7e.png)
With `overflow-x-auto` the scroll will be visible only if there are many verifiers:
![image](https://user-images.githubusercontent.com/140053/87954911-ac7d3700-cab5-11ea-96af-79c2931263ea.png)
And there will be no scroll for the small amount of verifiers:
![image](https://user-images.githubusercontent.com/140053/87955091-eea67880-cab5-11ea-9172-95b5ce6bd071.png)

Reply:
Also, I found that we can't access the Objectives page (the link like `http://localhost:3000/community/<COMMUNITY_NAME>/objectives`).

There was an "Edit" button on pages with URL `http://localhost:3000/community/<COMMUNITY_NAME>`, but it has been disappeared:
![image](https://user-images.githubusercontent.com/140053/87956876-37f7c780-cab8-11ea-9a59-66a36379159a.png)

I guess the button was lost in [this commit](https://github.com/cambiatus/frontend/commit/7a938bcfe795ba9646c92a219681207cbf5e7e23#diff-6038d0b94f3cc929f405c94e8b4f5864R160) while working on Community features.

app.cambiatus.io/ still have this button, but `master` branch not. I guess that's because we haven't updated dev server for some time.

Reply:
@amiskov That is correct. This button has moved to a cog icon in the dashboard. Please note that this cog icon link should only be available if you are the community creator

Reply:
@lucca65 thanks for the clarification! Missed that change...